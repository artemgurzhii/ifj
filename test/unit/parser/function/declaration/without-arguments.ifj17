declare function foo() as integer
