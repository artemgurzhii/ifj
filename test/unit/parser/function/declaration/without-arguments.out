(declare
  (function foo -> (id integer)))
